### DISCUSSION TITLE
Delete/hide buttons on main window

### DISCUSSION THREAD

**Discussion by faceliftmike** [AntonyCorbett/OnlyM#409] [Q&A]
Hi,
I have installed the latest version, the fixed size window will be a huge help on zoom thank you.
Not sure if I am missing something but the delete/hide buttons appear to be missing from the main window now?  The only way to delete files is directly in the image folder?

**Comment by AntonyCorbett** [comment]
Is **Media Files => Enable command panel** turned on?

**Reply by faceliftmike** [reply]
Thank you. No it wasnt, I missed that. Must have been turned off in the course of the update.
